frameworks/bitcoin-move/sources/bitcoin.move

```diff
@@ -87,11 +93,61 @@ module bitcoin_move::bitcoin{
 
         let block = bcs::from_bytes<Block>(block_bytes);
         process_txs(btc_block_store, &block, block_height);
-        let block_header = types::header(&block);
+        btc_block_store.reorg_status = option::some(string::utf8(b"active")); // reorg status is always active for newest block
```

Comment:
We just lazy processed the Bitcoin block and waited about 3 blocks. Otherwise the utxo status is hard to progress.

Reply:
You mean it's already waited and delayed for 3 blocks from the network and there needs to be modified according to that?

Reply:
Delay for X blocks to progress UTXO in the Move contract.

Reply:
> Delay for X blocks to progress UTXO in the Move contract.

Okay. I'll do delay 2 blocks for the bitcoin relayer on `submit_new_block` to the Move light client for processing the UTXO for the possible chain reorg, since 2 blocks is the most common scenario for the same txs happening at the same time.

Reply:
1. Give a config to the delay blocks.
2. Do not change the executor and delay in the Move module.
3. If reorg blocks > config delay blocks, record a status, make the chain unavailable, and stop executing new transactions. We need to manual handle it. 

If you have any questions, start a thread in Discord.